[2019-05-01T15:03:18,582][ERROR][logstash.javapipeline    ] A plugin had an unrecoverable error. Will restart this plugin.
  Pipeline_id:main
  Plugin: <LogStash::Inputs::Jdbc jdbc_user=>"root", jdbc_paging_enabled=>true, jdbc_password=><password>, statement=>"select title from information", jdbc_page_size=>10000, jdbc_pool_timeout=>120, jdbc_driver_library=>"C:/elastic/logstash/lib/mysql-connector-java-8.0.16.jar", jdbc_connection_string=>"jdbc:mysql://127.0.0.1:3306/youtube?useUnicode=true&characterEncoding=utf8&serverTimezone=Asia/Seoul", id=>"827c7c69376435b97a085215a4df741aa87729174960577ed7c20c4c99db0d39", jdbc_driver_class=>"com.mysql.jdbc.Driver", enable_metric=>true, codec=><LogStash::Codecs::Plain id=>"plain_a27e1002-f559-48a6-b668-5340ba293aaf", enable_metric=>true, charset=>"UTF-8">, jdbc_validate_connection=>false, jdbc_validation_timeout=>3600, sql_log_level=>"info", connection_retry_attempts=>1, connection_retry_attempts_wait_time=>0.5, parameters=>{"sql_last_value"=>1970-01-01 00:00:00 UTC}, last_run_metadata_path=>"C:\\Users\\zlslsp54/.logstash_jdbc_last_run", use_column_value=>false, tracking_column_type=>"numeric", clean_run=>false, record_last_run=>true, lowercase_column_names=>true>
  Error: com.mysql.jdbc.Driver not loaded. Are you sure you've included the correct jdbc driver in :jdbc_driver_library?
  Exception: LogStash::ConfigurationError
  Stack: C:/elastic/logstash/vendor/bundle/jruby/2.5.0/gems/logstash-input-jdbc-4.3.13/lib/logstash/plugin_mixins/jdbc/jdbc.rb:163:in `open_jdbc_connection'
C:/elastic/logstash/vendor/bundle/jruby/2.5.0/gems/logstash-input-jdbc-4.3.13/lib/logstash/plugin_mixins/jdbc/jdbc.rb:221:in `execute_statement'
C:/elastic/logstash/vendor/bundle/jruby/2.5.0/gems/logstash-input-jdbc-4.3.13/lib/logstash/inputs/jdbc.rb:277:in `execute_query'
C:/elastic/logstash/vendor/bundle/jruby/2.5.0/gems/logstash-input-jdbc-4.3.13/lib/logstash/inputs/jdbc.rb:263:in `run'
C:/elastic/logstash/logstash-core/lib/logstash/java_pipeline.rb:297:in `inputworker'
C:/elastic/logstash/logstash-core/lib/logstash/java_pipeline.rb:290:in `block in start_input'
[2019-05-01T15:03:19,217][WARN ][logstash.runner          ] SIGINT received. Shutting down.
[2019-05-01T15:03:19,590][ERROR][logstash.inputs.jdbc     ] Failed to load C:/elastic/logstash/lib/mysql-connector-java-8.0.16.jar {:exception=>#<TypeError: failed to coerce jdk.internal.loader.ClassLoaders$AppClassLoader to java.net.URLClassLoader>}
